server/__init__.py

```diff
@@ -89,9 +89,19 @@ def register_routes_custom_dc(app):
 
 
 def register_routes_stanford_dc(app):
-  # Apply blueprints specific to Stanford DC
+  # Install blueprints specific to Stanford DC
   from routes import (disasters)
+  from routes.api import (disaster_api)
   app.register_blueprint(disasters.bp)
+  app.register_blueprint(disaster_api.bp)
+
+  # load disaster dashboard data from GCS
+  if os.environ.get('FLASK_ENV') in [
+      'autopush', 'local', 'dev', 'local-stanford', 'stanford'
```

Comment:
Since this is part of register_routes_stanford_dc, we don't need this additional `if`.

Reply:
Fixed!